cli/cmdcontext/cmdcontext.go

```diff
@@ -61,6 +62,72 @@ func (tntCli *TarantoolCli) GetVersion() (version.Version, error) {
 	return tntCli.version, nil
 }
 
+// TtCli describes tt executable.
+type TtCli struct {
```

Comment:
Why do we need this struct? We already have `version.GetVersion` to get our own version. And we may have a function to get a version of the specified binary. This TtCli object is not used by any code outside of the `install` package.

Reply:
Deleted struct and leaved function `GetTtVersion`.